### ISSUE TITLE
Swiped capslock and left ctrl

### ISSUE THREAD

**Issue by pmtremblay** [Xigtun/xps-9570-mojave#11]
Bonjour,

You wrote: "I have swipe the capslock and left ctrl, replace a VoodooPS2Controller.kext can restore"
I don't totally understand your sentence. Personally I would prefer the reverse (keeping caps lock as is and left ctrl as is). Sometimes cap lock led stays forever and it confuses me.

What should I do to revert this back? 

Thanks.

**Comment by Kevincible** [comment]
to swap them back, see [OS-X-Voodoo-PS2-Controller](https://github.com/RehabMan/OS-X-Voodoo-PS2-Controller) for more info

**Comment by pmtremblay** [comment]
Thanks Kevincible but I don't understand what I would need to do. I tried the internet but it's still not helping me. I would need more details. 
I tried to look in EFI repo from here to see if I could find the file SSDT-Swap-LeftControlCapsLock.dsl but it is not listed under EFI\CLOVER\ACPI\patched but it wasn't. I looked in config.plist but could not find it either.
What am I missing here?

Thanks.
